subsystems/DiscordRPC/src/main/java/org/terasology/engine/subsystem/discordrpc/DiscordRPCSystem.java

```diff
@@ -13,9 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
```

Comment:
Let's update all the license headers in the extracted subsystem to use the shorter two-line license.